Components/Nav/nav.html

```diff
@@ -0,0 +1,114 @@
+            </div>
+            <div class="content-heading">
+                <p><b>Components</b></p>
+            </div>
+            <div class="sub-content">
+                <ul class="item-list">
+                  <li> <a href="/Components/Avatar/avatar.html"> Avatar</a></li>
+                  <li><a href="/Components/Badges/badges.html">Badges</a></li>
+                  <li><a href="/Components/Alerts/alerts.html">Alerts</a></li>
+                  <li><a href="/Components/Buttons/buttons.html">Buttons</a></li>
+                  <li><a href="/Components/Cards/cards.html">Cards</a></li>
+                  <li><a href="/Components/image/image.html">Images</a></li>
+                  <li><a href="/Components/Input/input.html">Input</a></li>
+                  <li><a href="/Components/List/lists.html">List</a></li>
+                  <li><a href="/Components/Ratings/ratings.html">Ratings</a></li>
+                  <li><a href="/Components/Snackbar/snackbar.html">Snackbar</a></li>
+                  <li><a href="/Components/Chips/chips.html">Chips</a></li>
+                  <li><a href="/Components/Nav/nav.html">Navigation Bar</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="content-bar">
+            <div class="content-title">Navigation Bar</div>
+        <div class="content-subtitle border_to_content">
+            Having easy-to-use navigation is important for any web site.
+            With CSS you can transform boring HTML menus into good-looking navigation bars.
+        </div>
+        <pre>
+            <code class="plaintext">
+                &lt;nav&gt;
+                    &lt;div class="nav-bar"&gt;
+                        &lt;div class="left-navbar"&gt;
+                        &lt;h2&gt;&lt;a href="/"&gt;    &lt;span class="primary-highlight"&gt;Untz&lt;/span&gt;Untz!&lt;/a&gt;&lt;/h2&gt;
+                        &lt;/div&gt;
+                        &lt;div class="search-bar flex_r"&gt;
+                        &lt;i class="search-icon fa-solid fa-magnifying-glass"&gt;&lt;/i&gt;
+                        &lt;input
+                            type="search"
+                            placeholder="Search for products, brands & more"
+                        /&gt;
+                        &lt;/div&gt;
+                    &lt;div class="right-navbar"&gt;
+                        &lt;a href="/"&gt;Login&lt;/a&gt;
+                        &lt;a href="/"&gt;Signup&lt;/a&gt;
+                        &lt;a href="/"&gt;Explore&lt;/a&gt;
+                    &lt;/div&gt;
+                    &lt;/div&gt;
+              &lt;/nav&gt;
+            </code>
+        </pre>
+        <div class="content-example">
+            <nav>
+                <div class="nav-bar">
+                    <div class="left-navbar">
+                      <h2><a href="/"> Title </a></h2>
+                    </div>
+                    <div class="search-bar flex_r">
+                      <input
```

Comment:
Add some styling to input element as currently its completely blending with the nav background